lib/logstash/inputs/gelf.rb

```diff
@@ -157,7 +159,7 @@ def self.from_json_parse(json)
   # legacy_parse uses the LogStash::Json class to deserialize json
   def self.legacy_parse(json)
     o = LogStash::Json.load(json)
-    LogStash::Event.new(o) if o
+    LogStash::Event.new(o)
```

Comment:
ah ok, so I don't understand your previous comment «Also this mean the code is broken in gelf if this case happen, because there is no nil check after parsing the event.»

so, is there a check for nil or not, is it broken or not a this point?